File: examples/sdk/java/Discover.java
public final class Discover {
    public static void main(String[] args) {
        try (Lnd.Client client = new Lnd.Client("http://127.0.0.1:8765", "dev-token")) {
            String nodes = client.discoverJson(
                new Lnd.DiscoveryFilter("office-a")
                    .withService("_demo._tcp")
                    .addTag("stable")
            );
            System.out.println(nodes);
        }
    }
}

File: examples/sdk/cpp/discover.cpp
#include <iostream>

#include "lnd.hpp"

int main() {
  lnd::Client client("http://127.0.0.1:8765", "dev-token");
  auto nodes = client.discover_json(
      lnd::DiscoveryFilter("office-a").with_service("_demo._tcp").add_tag("stable"));
  std::cout << nodes << std::endl;
  return 0;
}
